src/web-push-lib.js

```diff
@@ -252,6 +245,14 @@ WebPushLib.prototype.generateRequestDetails =
           requestDetails.headers['Crypto-Key'] = vapidHeaders['Crypto-Key'];
         }
       }
+    } else if (isGCM || isFCM) {
```

Comment:
Are you sure this works with FCM?

Reply:
Yes it does work with FCM, that is why I'm trying to add as a supported webpush endpoint.